backend/api/Controllers/DocumentController.cs

```diff
@@ -40,12 +49,12 @@ public DocumentController(IDocumentService documentService)
         /// </summary>
         [HttpGet("types")]
         [HasPermission(Permissions.PropertyAdd)]
```

Comment:
question - is this the right permission? PropertyAdd ?

Reply:
I don't think so, we don't have a document permission at the moment